posthog/models/experiment.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
+    name: models.CharField = models.CharField(max_length=400)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+
+    # Filters define the target metric of an Experiment
+    filters: models.JSONField = models.JSONField(default=dict)
```

Comment:
after talking to @Twixes , perhaps we can add a key that connects an experiment and insight? So that the frontend doesn't have to create and manipulate a new one every time

Reply:
nvm, we don't have to do this i think